src/content/comments/core-concepts/configure.mdx

```diff
@@ -68,3 +70,26 @@ Comments.Configure({
   onClickThread: (id) => console.log('Thread clicked', id),
 })
 ```
+
+## useLegacyWrapping
+
+<Callout title="Warning" variant="warning">
+  The new wrapping mechanism is using a different schema for block thread nodes, which is not
+  compatible with the legacy wrapping mechanism. If you are using the new wrapping mechanism, you
+  need to set this option to `false`. If this is set to `false` without mapping existing thread
+  nodes to the new schema, the threads content will be stripped from the document.
+</Callout>
+
+A boolean option that controls whether to use the legacy wrapping mechanism for multi-line comments. This is required to support backwards compatibility with existing comments and will
+be removed in the future. The default value is `true`.
+
+The new wrapping mechanism is more flexible, allowing to wrap content more precise and supports mixed wrapping of inline and block nodes.
+
+Default: `true`
+
+```js
+Comments.configure({
+  // enable new flexible block wrapping
+  useLegacyWrapping: false,
+})
+```
```

Comment:
```suggestion
<Callout title="Warning" variant="warning">
  The new wrapping mechanism is uses a different schema for threads on block nodes, which is not
  compatible with the previous wrapping behavior. If this is set to `false` without mapping existing thread
  nodes to the new schema, the threads content will be stripped from the document.
</Callout>

A boolean option that controls whether to use the legacy wrapping mechanism for multi-line comments. We suggest for new implementations to set this to `false`, and existing integrations can stay on the previous behavior.  This is only required for backwards compatibility with existing comments, and it will
be removed in the future.

The new wrapping mechanism is more flexible, allowing to wrap content more precise and supports mixed wrapping of inline and block nodes.

**Default:** `true`

```js
Comments.configure({
  // enable new flexible block wrapping
  useLegacyWrapping: false,
})
```
```

Reply:
Thanks!